pkg/lib/pb/model/protos/models.proto

```diff
@@ -962,8 +964,11 @@ message Notification {
         Export export = 8;
         GalleryImport galleryImport = 9;
         RequestToJoin requestToJoin = 10;
+        RequestResponse requestResponse = 11;
+        ParticipantRequestApproved participantRequestApproved = 12;
     }
     string space = 7;
+    string acl = 13;
```

Comment:
what is the purpose of this field?

Reply:
So we can use for `lastNotificationByAcl` in notification service for indexing purpose. So we don't need to iterate each time from the beginning `Acl.Records()`

Reply:
maybe name it `AclHeadId`? And in other places too

Reply:
Do you mean rename `acl` field in `Notificaion` model?

Reply:
yes, it's more descriptive